retrospect/20250204.md

```
@@ -0,0 +1,39 @@
+## 리뷰
+  ### 데일리 스크럼
+
+  ### AWS
+  - Region, AZ(물리적으로는 여러 데이터센터가 밀접하게 합쳐진 개념), Edge Location(캐싱 많이 필요한 static file, Lambda Edge.) 설명
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)